trainee_template/react_into_trainee.amlg

```diff
@@ -475,7 +477,7 @@
 					))
 				)
 
-				(if mda_robust_permutation
+				(if mda_robust_permutationy
```

Comment:
was this intentional? 'mda_robust_permutationy'

Reply:
Cat